tlp-func-base.in

```diff
@@ -357,6 +359,37 @@ read_defaults () { # read config file
     fi
 }
 
+read_customization () { # read customized config file
+    for i in $(find $CUSTOMIZECONFDIR -type f | sort); do
+        . $i
+    done
+    return 0
+}
+
+read_confs () { # read all config files
+    if read_defaults && read_customization;then
+        return 0
+    else
+        return $resule
```

Comment:
Where does `$resule` come from?

Reply:
What does `$resule` stand for?

Reply:
looks it's a typo there, will revise it.